model/src/main/scala/cromwell/pipeline/model/wrapper/Name.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.pipeline.model.wrapper
+
+import cats.data.Validated.{ Invalid, Valid }
+import cats.data.{ NonEmptyChain, Validated }
+import cromwell.pipeline.model.validator.Wrapped
+import play.api.libs.json.{ Format, JsError, JsResult, JsSuccess, JsValue, Json }
+import cats.implicits.catsStdShowForString
+
+final class Name private (override val unwrap: String) extends AnyVal with Wrapped[String]
+
+object Name extends Wrapped.Companion {
+  type Type = String
+  type Wrapper = Name
+  type Error = String
+  implicit lazy val nameFormat: Format[Name] = new Format[Name] {
+    override def reads(json: JsValue): JsResult[Name] = Name.from(json.as[String]) match {
+      case Valid(x)   => JsSuccess(Name(x))
+      case Invalid(e) => JsError(e.head)
```

Comment:
I think we must keep all errors, not only the first error.

Reply:
Done